### DISCUSSION TITLE
Not received my student pack benefits

### DISCUSSION THREAD

**Discussion by anajibi** [community/community#139131] [GitHub Education]
### Hi everyone,

Hey,
I was approved for GitHub student pack 6 days ago and have not yet received my benefits, what should I do?

**Comment by davevad93** [comment]
Hi @anajibi ,

Don't worry you don't have to do anything, just wait. The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @anajibi was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by matheuslpiresm** [comment]
I was also approved and I'm waiting, the last update was:

Congrats! Your request was approved and your benefits will be available!
